samples/msgext-ai-doc-compliance-checker/README.md

```diff
@@ -0,0 +1,130 @@
+---
+page_type: sample
+description: This sample demonstrates integrating Azure Open AI into a Teams message extension to check a contract proposal's compliance with a policy checklist. It works with Teams Toolkit for Visual Studio Code and JavaScript and can be used in Copilot for Microsoft 365.
+products:
+- office-teams
+- copilot-m365
+- azure open-ai
+languages:
+- javascript
+---
+
+# Compliance Checker using Azure Open AI
+
+This sample uses Azure Open AI to compare proposal documents uploaded to an Azure Blob storage with a policy guideline that is also uploaded in the blob.
+ 
+The solution makes calls to Azure Open AI to assess compliance against each of the line items mentioned in the policy guideline, and returns a result that is formatted and displayed as an adaptive card. 
+ 
+The sample can also be used as a Copilot Plug-in.
+
+## Interaction with app
+
+ ![msgext-doc-compliance-checkerGif](Images/msgext-doc-compliance-checkerGif.gif)
+
+## Try It Yourself - Experience the app in your Microsoft Teams and Copilot client
+Please find below demo manifest which is deployed on Microsoft Azure and you can try it yourself by uploading the app package (.zip file link below) to your teams and/or as a personal app. (Sideloading must be enabled for your tenant, [see steps here](https://docs.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading)).
+
+**Compliance Checker sample app:** [Manifest](/samples/msgext-ai-doc-compliance-checker/demo-manifest/AIComplianceChecker.zip)
+
+## Prerequisites
+
+- [Node.js 18.x](https://nodejs.org/download/release/v18.18.2/)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Blob Storage](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal)
+- [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)
+- You will need a Microsoft work or school account with [permissions to upload custom Teams applications](https://learn.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading). The account will also need a Microsoft Copilot for Microsoft 365 license to use the extension in Copilot.
+- [Azure Open AI](https://learn.microsoft.com/en-us/azure/ai-services/openai/quickstart?tabs=command-line&pivots=programming-language-studio)
+
+## Setup and use the sample
+
+1) **Create Azure Blob Storage:**
```

Comment:
Step is missing to create Azure Open AI service and which model we need to deploy?

Refer this PR: https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/1335/files#diff-057c0397feadec74b5f5e1355db65d2c0a2729842749a0affc7dc757ea7c499dR79

Reply:
Fixed